app/views/logs/_form.html.erb

```diff
@@ -21,11 +21,13 @@
     <%= form.text_area :model %>
   </div>
 
-  <div class="control-group">
-    <%= form.label :service, class: 'control-group' %>
-    <div class="controls">
-      <%= collection_select( :logs, :service_id, Service.all, :id, :title, {}, { :multiple => false})%>
-    </div>
+  <div class="field">
+      <%= form.label :services, style: "display: block" %>
+      <%= hidden_field_tag "log[service_ids][]", nil%>
+    <% Service.all.each do |service|%>
```

Comment:
this seems fine. can you check if this would work?

https://makandracards.com/makandra/32147-rails-4-introduced-collection_check_boxes

the example from the links uses form_for, 
but as of rails 6, form_with acts as a form_for when a model i passed, so this should essentially work for this scenario

Reply:
tried this, seems not to work. will check further. Can we discuss this during the weekly meeting?